src/openrct2-ui/scripting/CustomListView.h

```diff
@@ -52,6 +52,7 @@ namespace OpenRCT2::Ui::Windows
 
     struct ListViewItem
     {
+        bool IsSeperator{};
```

Comment:
```suggestion
        bool IsSeparator{};
```

Guess it should read "separator" all across the PR?